### PR TITLE
founder -> crat in all languages

### PR DESCRIPTION
**Pull request by LegoFan4000**

### CODE REVIEW

**Comment by JohnFLewis** [comment]
This is good for the main page messages.

But the code still references "founders" and forms will still use the words "founder".

**Comment by JohnFLewis** [comment]
a bureaucrat doesn't exist in the sense of CreateWiki. People requesting wikis are requestors.

**Comment by LegoFan4000** [comment]
What would you like me to do then?

**Comment by JohnFLewis** [comment]
every instance of founder in the code and language files (exception the main page message) needs to changed to "requestor".

The use of "founder" is wrong through out including in the actual code.

**Comment by LegoFan4000** [comment]
@JohnFLewis Done

File: i18n/ro.json [hunk 1]
```diff
@@ -5,17 +5,17 @@
...
 	"createwiki-desc": "O extenție folositoare pentru crearea de wiki.",
     	"requestwiki-error-csrf": "<strong>Ne pare rău, wikiul dumneavoastra nu a putut fi creat din cauza unei piederi de sesiune de date.</strong>\n Vă rugăm să trimiteți din nou forumlarul. Daca nu merge [[Special:Userlogout|delogați-vă]] și logați-vă din nou.",
 	"createwiki-error-dbexists": "Un wiki cu acest nume de bază de date există deja.",
-	"createwiki-error-foundernonexistent": "Acest fondator există deja pe Miraheze Meta",
+	"createwiki-error-requesternonexistent": "Acest requester există deja pe Miraheze Meta",
```
Comment (Reception123) [inline i18n/ro.json]: replace all "requester" with "solicitant" in ro version

File: i18n/fr.json [hunk 2]
```diff
@@ -5,17 +5,17 @@
...
 	"createwiki-desc": "Un extention utile pour creér des wikis.",
         "requestwiki-error-csrf": "<strong>Desolé ce demande ne peut pas être enregisté.</strong>\nMerci de soummetre en core une fois. Si sa marche non plus, merci de [[Special:Userlogout|vous deconnecter]] et vous connecter a nouveau.",
 	"createwiki-error-dbexists": "Il y a deja un wiki avec ce nom",
-	"createwiki-error-foundernonexistent": "Le fondateur n'exsiste pas sur Miraheze Meta.",
+	"createwiki-error-requesternonexistent": "Le requester n'exsiste pas sur Miraheze Meta.",
```
Comment (Reception123) [inline i18n/fr.json]: replace "requester" with "demandeur"

File: i18n/de.json [hunk 3]
```diff
@@ -6,17 +6,17 @@
...
 	"createwiki-desc": "Extension zur einfachen Erstellung einer Wiki.",
         "requestwiki-error-csrf": "<strong>Sorry, diese Wiki konnte nicht erstellt werden wegen misteriösem Datenverlust.</strong>\nBitte sende Sie das Formular erneut. Sollte das nicht funktionieren, [[Special:Userlogout|Loggen Sie sich aus]] und dann wieder ein.",
 	"createwiki-error-dbexists": "Eine Wiki mit dem Database-namen besteht schon.",
-	"createwiki-error-foundernonexistent": "Der Gründer ist nicht bei Miraheze registriert.",
+	"createwiki-error-requesternonexistent": "Der Requester ist nicht bei Miraheze registriert.",
```
Comment (Reception123) [inline i18n/de.json]: replace "requester" with "anfrager"

**Pull request merged**